### PR TITLE
Add better Uptime Annoucement for System issues

### PR DESCRIPTION
**Pull request by Cliftonz**
### What change does this PR introduce?

This pr adds an integration for the web deployment to have announcements from better uptime be propagated to any user using Novu cloud.

### Why was this change needed?

This change allows us to communicate to users in the event of a failure.

### Other information (Screenshots)

image

### CODE REVIEW

**Review by BiswaViraj — APPROVED** [review]

**Comment by jainpawan21** [comment]
@Cliftonz is .idea directory files needed for this change?

**Comment by Cliftonz** [comment]
Yes, this tells Jetbrains to download this library for IntelliSense.

**Comment by Cliftonz** [comment]
> I am not familiar with uptime announcements, will it send toast announcements as in your screenshot? Additionally what announcements it includes? Will the different environments will be separated, meaning US users won't get EU announcements and announcements and vice versa?

Yes, but it will only do it if I manually put it in better uptime. This would not happen automatically.

No, it will be a catch all. Thus if we are having an issue for US we will need to make sure that is put in the announcement.

image

File: apps/web/public/index.html [hunk 1]
```diff
@@ -40,5 +40,11 @@
       To begin the development, run `npm start` or `yarn start`.
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
+    <script
```
Comment (scopsy) [inline apps/web/public/index.html:43]: @Cliftonz this should only be included for cloud based hosting, not self hosted

File: apps/web/public/index.html [hunk 2]
```diff
@@ -23,28 +23,21 @@
...
-      To begin the development, run `npm start` or `yarn start`.
-      To create a production bundle, use `npm run build` or `yarn build`.
-    -->
-    <script
-      src="https://uptime.betterstack.com/widgets/announcement.js"
-      data-id="144175"
-      async="async"
-      type="text/javascript"
-    ></script>
+    <% if ( process.env.REACT_APP_NOVU_MANAGED_SERVICE === 'true' ) { %>
```
Comment (djabarovgeorge) [inline apps/web/public/index.html:33]: @Cliftonz I know you already merged the PR but maybe we can reuse REACT_APP_DOCKER_HOSTED_ENV here so we won't have 2 environment variables that are similar to manage.

**Pull request merged**